packages/uswds-core/src/styles/mixins/typography/usa-prose.scss

```diff
@@ -5,58 +5,3 @@
 @use "./headings" as *;
 @use "./typeset" as *;
 
-@mixin usa-prose-link {
```

Comment:
Can we delete this file and all references to it? It seems like it isn't adding any styles anymore.

Reply:
This seems to be the only `usa-prose` mixin being used. So if we move the styles to the `usa-link-style` where it's being called we can get rid of the mixin all together